.github/workflows/builds.yml

```diff
@@ -39,6 +39,11 @@ jobs:
             cxx_standard: "20"
             options: -DTRACCC_BUILD_CUDA=TRUE -DCMAKE_CUDA_FLAGS="-std=c++20"
             run_tests: false
+          - name: CUDA-double
```

Comment:
As inconvenient it is, the name should be `CUDA`, not `CUDA-double`. :thinking: That name is something that we can make some decisions about. Even if we currently don't. (This is why all of the SYCL tests are named `SYCL` as well.)

Reply:
OK Fixed now